aries-site/src/examples/components/layouts/HeaderExample.js

```diff
@@ -8,41 +8,40 @@ export const HeaderExample = () => {
   const [focused, setFocused] = useState(false);
 
   return (
-    <Nav
-      title="Design System"
-      collapse={false}
-      background="background-front"
-      logoOnly={size === 'small' && focused}
-      pad={{ horizontal: 'small', vertical: 'small' }}
-    >
-      <Box
-        align="center"
-        background={
-          size !== 'small' || focused ? 'background-contrast' : undefined
-        }
-        direction="row"
-        justify="between"
-        onClick={() => setFocused(true)}
-        onFocus={() => setFocused(true)}
-        pad={{ right: 'small' }}
-        round="small"
-        width={size !== 'small' || focused ? 'medium' : undefined}
-      >
-        {size !== 'small' || focused ? (
-          <TextInput placeholder="Search HPE Design System" plain />
-        ) : (
-          undefined
-        )}
+      <Header fill="horizontal" pad="none" background='background-front'>
+        <AppIdentity
+          brand="hpe"
+          logoOnly={size === 'small'}
```

Comment:
I think we'd still want the HPE Design System text to appear unless the TextInput is focused, this can be in a separate PR though if you'd prefer.
<img width="577" alt="Screen Shot 2020-03-06 at 10 18 50 AM" src="https://user-images.githubusercontent.com/12522275/76110852-1cba6e80-5f94-11ea-8f25-607686b6d1ef.png">

Reply:
yes I agree I think since Ill have to look at the logic of how that is hidden in the search component so I think this will be better to put in another PR I did add this to issue #445